components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1458,7 +1458,7 @@ void RewardsServiceImpl::SetRewardsMainEnabled(bool enabled) {
 
   if (!enabled) {
     RecordRewardsDisabledForSomeMetrics();
-    StopLedger();
+    StopLedger(base::BindOnce([](ledger::Result) {}));
```

Comment:
you can just do `StopLedger(base::DoNothing())`

Reply:
Really helpful! Fixed.